tests/iostability/test_iostability_degraded_path.py

```diff
@@ -68,23 +68,41 @@ def setup_class(cls):
         cls.s3t_obj = S3TestLib()
         cls.iolib = IOStabilityLib()
         cls.test_completed = False
+        cls.duration_in_days = os.getenv("DURATION_OF_TEST_IN_DAYS",
+                                         cls.test_cfg['happy_path_duration_days'])
+
+        cls.clients = os.getenv("CLIENT_SESSIONS_PER_WORKER_NODE", 0)
+        if cls.clients == 0:
+            if cls.setup_type == 'HW':
```

Comment:
@priyankaborawake, is it possible that there will be any other value other than hw and vm? Human error? In that case, default case handling would be needed.